README.md

```diff
@@ -28,15 +31,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+data-ng-click allows html validation. ng-click simply connects it to an angular model/action
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+A - ng-app should always be inserted in the html file so that angular modules will be available to the entire application
```

Comment:
I think you're right. I want to mention that it doesn't always have to be in the `<html>` tag. Essentially is wherever you want that modules functionality to exist within.